staging/kubefed/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+description: KubeFed helm chart
+name: kubefed
+version: 0.5.2
```

Comment:
Please don't match the chart version to the app version. These should be expected to diverge and when they do it causes confusion.

This is true for all Chart.yaml in this PR.